airbyte-webapp/src/views/Connector/ServiceForm/ServiceForm.test.tsx

```diff
@@ -304,6 +304,7 @@ describe("Service Form", () => {
       const submit = container.querySelector("button[type='submit']");
       await waitFor(() => userEvent.click(submit!));
 
+      // @ts-expect-error typed unknown, okay in test file
```

Comment:
:information_source: `result.connectionConfiguration` is explicitly typed as `unknown` and was like that already before the upgrade. I have no idea how the previous typescript version did not catch this :thinking: Anyway, since it's a test file, simply ignoring this for now.